cmd/explaintest/r/partition_pruning.result

```diff
@@ -2276,7 +2276,7 @@ TableReader	250.00	root	partition:dual	data:Selection
   └─TableFullScan	10000.00	cop[tikv]	table:t1	keep order:false, stats:pseudo
 explain format = 'brief' select * from t2 where a > 0xFE AND a <= 0xFF;
 id	estRows	task	access object	operator info
-TableReader	250.00	root	partition:all	data:Selection
+TableReader	250.00	root	partition:dual	data:Selection
```

Comment:
? It's shouldn't be `dual` here.

Reply:
This is the same with MySQL's behavior:

```
create table t2 (a int)	create table t2 (a int)
partition by range(a+0) (	partition by range(a+0) (
partition p0 values less than (64),	partition p0 values less than (64),
partition p1 values less than (128),	partition p1 values less than (128),
partition p2 values less than (255)	partition p2 values less than (255)
);
```

partition by (a + 0),  a > 0xFE AND a <= 0xFF, the only possible value is 0xFF(255), there is no partition for it, so the result is `dual`